ImperatorToCK3/Source/CK3/Titles/Title.cpp

```diff
@@ -232,6 +237,21 @@ void CK3::Title::trySetAdjectiveLoc(mappers::LocalizationMapper& localizationMap
 		Log(LogLevel::Warning) << titleName << " help with localization for adjective! " << imperatorCountry->getName() << "_adj?";
 }
 
+void CK3::Title::setRank() {
+	if (titleName.starts_with('b'))
+		rank = TitleRank::barony;
+	else if (titleName.starts_with('c'))
+		rank = TitleRank::county;
+	else if (titleName.starts_with('d'))
+		rank = TitleRank::duchy;
+	else if (titleName.starts_with('k'))
+		rank = TitleRank::kingdom;
+	else if (titleName.starts_with('e'))
+		rank = TitleRank::empire;
+	else
+		throw std::runtime_error("Title " + titleName + ": unknown rank!");
```

Comment:
There are no xx titles in what you're loading? The merc companies and such? If this is for 00_landed_titles.txt then all is well.

Reply:
Indeed, the CK3 titles are read from common/landed_titles